resources/views/container/layouts/metric.blade.php

```diff
@@ -8,12 +8,16 @@
                 <small class="text-muted block mb-1">{{ __($key) }}</small>
                 <p class="h4 mb-1 text-black font-thin">{{ $metric['keyValue'] }}</p>
 
-                @if((float)$metric['keyDiff'] < 0)
-                    <small class="text-danger">{{ $metric['keyDiff'] }} % <i class="icon-arrow-down"></i></small>
-                @else
-                    <small class="text-success">{{ $metric['keyDiff'] }} % <i class="icon-arrow-up"></i></small>
+                @if(isset($metric['keyDiff']))
```

Comment:
it can be written easier
‘’’php
@isset($records)
 // $records is defined and is not null...
@endisset
‘’’

Reply:
Yes, forgot about this ability. Agree.